### PR TITLE
Skip smmp tests for p5 instances

### PR DESCRIPTION
**Pull request by kace**
*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Tests run

**NOTE: By default, docker builds are disabled. In order to build your container, please update dlc_developer_config.toml and specify the framework to build in "build_frameworks"**
- [ ] I have run builds/tests on commit <INSERT COMMIT ID> for my changes.

**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

### Formatting
- [ ] I have run `black -l 100` on my code (formatting tool: https://black.readthedocs.io/en/stable/getting_started.html)

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/pytorch/training/conftest.py [hunk 1]
```diff
@@ -30,13 +30,10 @@
 from . import get_efa_test_instance_type
 
 from .utils import get_ecr_registry, NightlyFeatureLabel, is_nightly_context
-from .integration import (
-    get_framework_and_version_from_tag,
-    get_cuda_version_from_tag,
-    get_processor_from_image_uri,
```
Comment (shantanutrip) [inline test/sagemaker_tests/pytorch/training/conftest.py:36]: Is this not being used in this `conftest` anymore?

File: test/sagemaker_tests/pytorch/training/conftest.py [hunk 2]
```diff
@@ -30,13 +30,10 @@
...
-)
 from .utils.image_utils import build_base_image, are_fixture_labels_enabled
 
+from test.test_utils import get_cuda_version_from_tag, get_framework_and_version_from_tag
```
Comment (shantanutrip) [inline test/sagemaker_tests/pytorch/training/conftest.py:35]: Did we run SM local tests on the PR? I would suggest running a test. One thing that can be observed in this SM test suite is that we do not use anything from the generic `test_utils` file i.e. `test.test_utils`. 

SM local tests have a different execution style in which we make a tar of the SM test suite: https://github.com/aws/deep-learning-containers/blob/master/test/testrunner.py#L62-L70.

Thus, we might want to test and proceed accordingly.

File: test/sagemaker_tests/pytorch/training/conftest.py (hunk 1 above)
Comment (kace) [reply test/sagemaker_tests/pytorch/training/conftest.py:36]: This function was only being used in `skip_p5_tests()`. I replaced it with the `processor` fixture.

File: test/sagemaker_tests/pytorch/training/conftest.py (hunk 2 above)
Comment (kace) [reply test/sagemaker_tests/pytorch/training/conftest.py]: Forgot about that. Change reverted.

**Review by shantanutrip — APPROVED** [review]

**Pull request merged**